src/Api/OperationTypeDeprecationHelper.php

```diff
@@ -38,7 +38,7 @@ public static function getOperationType($operationType): string
         if (is_bool($operationType)) {
             @trigger_error('Using a boolean for the Operation Type is deprecated since API Platform 2.1 and will not be possible anymore in API Platform 3', E_USER_DEPRECATED);
 
-            $operationType = $operationType === true ? OperationType::COLLECTION : OperationType::ITEM;
+            $operationType = true === $operationType ? OperationType::COLLECTION : OperationType::ITEM;
```

Comment:
Useless is `true`! Fixed the problem, should you have! 😛

Reply:
FML

Reply:
It's funny though, in Javascript it's always better to strict compare (better performances) though on php it's the opposite...